styles/style.css

```diff
@@ -0,0 +1,236 @@
+.header{
+    background: white url(./fon.png);
+    background-size: cover;
+    font-family: 'Roboto', sans-serif;
+    width: 1440px;
+    height: 700px;
+}
+
+/* Menu */
+
+
+.header-nav{
+    position: absolute;
+    width: 1206px;
+    height: 80px;
+    margin-top: 24px;
+    margin-left: 117px;
+    background: #F3F4F6;
+    padding: 5px;
+    }
+.header-menu{
+    text-align: left;
+}
+
+.logo1{
+    display: inline-block;
+    width: 3px;
+    height: 20px;
+    background-color: white;
+    border-radius: 20px;
+    background: linear-gradient(to bottom right,#EC008C, #FC6767);
+    text-align: center;
+    margin-top: 30px;
+    margin-bottom: 30px;
+    margin-left: 32px;
+}
+.logo2{
+    display: inline-block;
+    width: 3px;
+    height: 16px;
+    background-color: white;
+    border-radius: 20px;
+    background: linear-gradient(to bottom right,#E65C00,#F9D423);
+    text-align: center;
+    margin-top: 32px;
+    margin-bottom: 32px;
+    margin-left: 5px;
+}
+.logo3{
+    display: inline-block;
+    width: 3px;
+    height: 12px;
+    background-color: white;
+    border-radius: 15px;
+    background: linear-gradient(to bottom right, #00D2FF, #3A7BD5);
+    text-align: center;
+    margin-top: 34px;
+    margin-bottom: 34px;
+    margin-left: 5px;
+}
+.logo4{
+    display: inline-block;
+    text-align: center;
+    position: absolute;
+    left: 71px;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 22px;
+    margin-top: 27px;
+    margin-bottom: 27px;
+    margin-left: 5px;
+}
+.logo5{
+    display: inline-block;
+    text-align: center;
+    position: absolute;
+    left: 114px;
+    font-weight: 700; 
+    font-style: normal;
+    font-size: 22px;
+    margin-top: 27px;
+    margin-bottom: 27px;
+    margin-right: 71px;
+}
+
+
+.header-menu_item{
+    display: inline-block;
+    position: absolute;
+    color: black;
+    font-size: 12px;
+    font-weight: 700px;
+    line-height: 18px;
+    text-transform: uppercase;
+    padding: 30px;
+}
+.about{
```

Comment:
remove all these styles with left, you can do it in an easier way, add separate block for navigation, and add right styles for it